src/settings/BatchGroupConfigurationSettings/constants.js

```diff
@@ -29,3 +29,8 @@ export const SCHEDULE_EXPORT_OPTIONS = Object.values(SCHEDULE_EXPORT).map(schedu
 }));
 
 export const RESULT_COUNT_INCREMENT = 10;
+
+export const EXPORT_FORMATS_HEADER_MAP = Object.values(EXPORT_FORMAT).reduce((acc, format) => ({
+  ...acc,
+  [format]: { 'Accept': format.toLowerCase() },
```

Comment:
But this doesn't change anything - you still have coupled constants for export config format and http headers. Please revisit my initial comment, where I asked to decouple them

Reply:
fixed